### PR TITLE
Fluent: load panel&indicator restyle

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/devextreme/js/ui/load_panel.d.ts [hunk 1]
```diff
@@ -125,6 +125,7 @@ export interface dxLoadPanelOptions extends dxOverlayOptions<dxLoadPanel> {
      * @docid
      * @default 90
      * @default 60 &for(Material)
+     * @default auto &for(Fluent)
```
Comment (EugeniyKiyashko) [inline packages/devextreme/js/ui/load_panel.d.ts]: Perhaps, we should write auto in single quotes.

Comment (EugeniyKiyashko) [reply packages/devextreme/js/ui/load_panel.d.ts]: @alexanderPolosatov Could you please resolve this case?

File: packages/devextreme/scss/widgets/fluent/loadIndicator/_index.scss [hunk 2]
```diff
@@ -69,29 +69,34 @@
...
+  height: 100%;
+  overflow: hidden;
+  box-sizing: border-box;
 
   .dx-loadindicator-segment-inner {
-    left: -100%;
-    border-left-color: transparent;
-    transform: rotate(129deg);
-    animation: dx-right-segment-rotation 1333ms cubic-bezier(0.4, 0, 0.2, 1) infinite both;
+    border-color: #D0E5FB;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/loadIndicator/_index.scss]: We shouldn't to use hard-coded value of color in our styles. Please, use base variables for this

File: packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js [hunk 3]
```diff
@@ -1270,11 +1270,11 @@ testComponentDefaults(LoadPanel,
         message: ''
     },
     function() {
-        this.origIsMaterialBased = themes.isMaterialBased;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js:1273]: What about to test Fluent() theme default options here(in this test file)?

File: packages/devextreme/js/ui/load_panel.d.ts (hunk 1 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/ui/load_panel.d.ts]: >Perhaps, we should write auto in single quotes.

Yes. Strings should be wrapped to quotes. @EugeniyKiyashko @iBat

File: packages/devextreme/scss/widgets/fluent/loadPanel/_index.scss [hunk 4]
```diff
@@ -10,13 +10,9 @@
...
+  border: none;
   background: $loadpanel-content-bg;
-  padding: 13px;
-  border-radius: 50%;
-  box-shadow: 0 6px 12px $loadpanel-content-shadow-color;
-}
-
-.dx-loadpanel-message {
-  display: none;
+  padding: 12px 24px;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/loadPanel/_index.scss]: Let's extract hard-cored values to consts.

Comment (EugeniyKiyashko) [reply packages/devextreme/scss/widgets/fluent/loadPanel/_index.scss]: And border-radius value too

File: packages/devextreme/scss/widgets/fluent/loadIndicator/_colors.scss [hunk 5]
```diff
@@ -9,3 +9,4 @@
 * $type color
 */
 $load-indicator-segment-bg-color: $base-accent !default;
+$load-indicator-inner-segment-border-color: #D0E5FB !default;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/loadIndicator/_colors.scss]: What about dark theme? The colors of segments are incorrect there. Also, in Light theme the color does not match the color of the guides

Comment (EugeniyKiyashko) [reply packages/devextreme/scss/widgets/fluent/loadIndicator/_colors.scss]: Fixed in https://github.com/DevExpress/DevExtreme/pull/25914/commits/b7809f549dc34382eddd031f028ffdbd38857d0c

**Review by alexanderPolosatov — DISMISSED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**